web/client/utils/LayersUtils.js

```diff
@@ -486,24 +485,38 @@ const LayersUtils = {
         layer.credits ? { credits: layer.credits } : {});
     },
     /**
-    * default regex rule for searching for a /geoserver/ string in a url
+    * default constant regex rule for searching for a /geoserver/ string in a url
     */
     REG_GEOSERVER_RULE,
```

Comment:
this is out of synch and seems not being used anywhere in github: [see](https://github.com/search?q=REG_GEOSERVER_RULE&type=Code)